DependencyInjection/RouteEnhancerCompilerPass.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ArticleBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+class RouteEnhancerCompilerPass implements CompilerPassInterface
```

Comment:
add also an issue in sulu that we have to add the compiler there for 2.0 (reoute-bundle)